### PR TITLE
Hide and show RSS links

### PR DESCRIPTION
**Pull request by luigifab**
### Description

This fix some bugs with RSS links in _Sales / Orders_, _Catalog / Products_ and _Catalog / Reviews_.
For example, when a RSS feed is disabled, it's not necessary to display the link.

### Manual testing scenario

You need at least, two store views. Set the following configuration for one of them (not for the default store):

config-rss-store

Go to _Catalog / Reviews_ (same for _Sales / Orders_ and _Catalog / Products_):

rss

The RSS link must be displayed only for the store where the link is enabled, and the link must use the current selected store.

I hope it's good, at the beginning I started to only add `Mage::helper('catalog')->isModuleEnabled('Mage_Rss')` and `Mage::getStoreConfigFlag('rss/xyz/xyz')`.

Edit: I forgot to commit CSV update.

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All automated tests passed successfully (all builds are green)
 - [x] Add yourself to contributors list

### CODE REVIEW

File: app/code/core/Mage/Adminhtml/Block/Review/Grid.php [hunk 1]
```diff
@@ -192,7 +192,20 @@ protected function _prepareColumns()
...
+        if (Mage::helper('catalog')->isModuleEnabled('Mage_Rss')) {
+            if ($this->getColumn('visible_in')) {
+                $filter = $this->getParam($this->getVarNameFilter());
+                if (!empty($filter)) {
+                    $filter = Mage::helper('adminhtml')->prepareFilterString($filter);
+                    if (!empty($filter['visible_in']))
+                        $storeId = $filter['visible_in'];
+                }
+            }
+            if (empty($storeId))
```
Comment (sreichel) [inline app/code/core/Mage/Adminhtml/Block/Review/Grid.php:204]: Reviewed a few commits ... can you please avoid `empty()` when not required and use more strict checks?

```
                $filter = $this->getParam($this->getVarNameFilter());
                if ($filter !== '') {
                    $filter = Mage::helper('adminhtml')->prepareFilterString($filter);
                    if (isset($filter['visible_in']))
                        $storeId = $filter['visible_in'];
                }
            }
            if ($storeId === null) # storeId should be defined above
```

Not tested.

File: app/code/core/Mage/Rss/etc/system.xml [hunk 2]
```diff
@@ -31,16 +31,16 @@
...
                     <show_in_default>1</show_in_default>
                     <show_in_website>1</show_in_website>
                     <show_in_store>1</show_in_store>
                     <fields>
                         <active translate="label">
-                            <label>Enable RSS</label>
+                            <label>Enable</label>
                             <frontend_type>select</frontend_type>
-                            <source_model>adminhtml/system_config_source_enabledisable</source_model>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
```
Comment (sreichel) [inline app/code/core/Mage/Rss/etc/system.xml:43]: Why?

Comment (luigifab) [reply app/code/core/Mage/Rss/etc/system.xml:43]: To reduce number of translations.

Before we have:
- RSS Config
- Enable RSS (with Enable / Disable)

Now we have:
- General
- Enable (with Yes / No)

Comment (sreichel) [reply app/code/core/Mage/Rss/etc/system.xml:43]: Mhh, i prefer `enabledisable` over `yesno` for this type of fields? Rly change it?

**Review by sreichel — CHANGES_REQUESTED** [review]
Seems you use wrong config pathes for admin grids.

Should be `admin_catalog` instead of `catalog`. See https://github.com/OpenMage/magento-lts/pull/2760#issuecomment-1368520272

**Comment by luigifab** [comment]
Closed for #2760.

**Pull request closed without merging**